pkg/config/config_template.yaml

```diff
@@ -67,17 +67,18 @@ api_key:
 ## @env DD_MIN_TLS_VERSION - string - optional - default: "tlsv1.0"
 ## This option defines the minimum TLS version that will be used when
 ## submitting data to the Datadog intake specified in "site" or "dd_url".
-## This parameter defaults to "tlsv1.0" (but see `force_tls_12`, below).
+## This parameter defaults to "tlsv1.2".
 ## Possible values are: tlsv1.0, tlsv1.1, tlsv1.2, tlsv1.3; values are case-
 ## insensitive.
 #
-# min_tls_version: "tlsv1.0"
+# min_tls_version: "tlsv1.2"
 
+## Deprecated - use `min_tls_version` instead
 ## @param force_tls_12 - boolean - optional - default: false
 ## @env DD_FORCE_TLS_12 - boolean - optional - default: false
-## Setting this option to "true" is equivalent to setting the default for
-## `min_tls_version` to `tlsv1.2`.  This parameter is deprecated in favor of
-## `min_tls_version`.
+## Setting this option to "true" forces the Agent to only use TLS 1.2 when
+## pushing data to the Datadog intake specified in "site" or "dd_url".
+## This parameter is deprecated in favor of `min_tls_version`.
 #
 # force_tls_12: false
```

Comment:
I this is ignored in the code we should remove it from the template and config.go.

Reply:
+1 to this on the doc side. We may want to leave it in the config.go but warn that it will be removed.

Reply:
I completely agree for `config.go` ! 
For the template, I saw that there was some option still marked as deprecated and I believe they are still here because it might have sense to keep them (at least for 1 or 2 releases) for information only. 

It's not a strong opinion at all here, I just wanted to point this out. If you can confirm that we can remove it from the template without any problem I'll be happy to do so !

Reply:
I kind of disagree. The configuration template is not a upgrade doc, the CHANGELOG is there for that. The option in the datadog.yaml marked as deprecated are still supported in the code but will be removed at some point. 

Here the setting was not deprecated but removed.